exporter/opentelemetry-exporter-otlp/src/opentelemetry/exporter/otlp/exporter.py

```diff
@@ -123,23 +131,42 @@ class OTLPExporterMixin(
         endpoint: OpenTelemetry Collector receiver endpoint
         credentials: ChannelCredentials object for server authentication
         metadata: Metadata to send when exporting
+        compression: Compression algorithm to be used in channel
     """
 
     def __init__(
         self,
         endpoint: str = "localhost:55680",
         credentials: ChannelCredentials = None,
         metadata: Optional[Tuple[Any]] = None,
+        compression: OTLPCompression = OTLPCompression.NO_COMPRESSION,
     ):
         super().__init__()
 
         self._metadata = metadata
         self._collector_span_kwargs = None
 
+        if compression == OTLPCompression.GZIP:
+            compression_algorithm = Compression.Gzip
+        else:
+            compression_str = os.environ.get(
+                "OTEL_EXPORTER_OTLP_COMPRESSION", None
+            )
```

Comment:
Try `Configuration().EXPORTER_OTLP_INSECURE` here

Reply:
Is there an advantage of using `Configuration().EXPORTER_OTLP_INSECURE` in this instance? I tried using it, and it broke all the OTLP tests. I took the structure for this part in getting the environment variable from the zipkin exporter `__init__.py` on line 116

Reply:
@aabmass Would `os.environ.get()` work in this instance without changing it to use `Configuration()`?